src/routes/webhook/discord/interaction/confess.ts

```diff
@@ -205,12 +220,29 @@ export async function handleConfess(
     channelId: Snowflake,
     authorId: Snowflake,
     [option, ...options]: InteractionApplicationCommandChatInputOption[],
+    resolved: Resolved | null,
+    permissions: bigint,
 ) {
-    strictEqual(options.length, 0);
+    strict(options.length <= 1);
     strictEqual(option?.type, InteractionApplicationCommandChatInputOptionType.String);
     strictEqual(option.name, 'content');
+
+    let attachment = null;
+    const attachments = Object.values(resolved?.attachments ?? {});
```

Comment:
Another nitpick: I dislike spawning sentinel objects out of thin air. I find this equivalent to be a little clearer on intent in my book.

```suggestion
    const attachments = typeof resolved === 'undefined'
        ? []
        : Object.values(resolved.attachments);
```